Add anchors for dotted circle and caron.alt

typiconman opened the pull request:
Adding anchor points for the dotted circle and the caron.alt glyph.
@yanone  please review. Not sure if caron.alt also needs some kind of GSUB mappings? I don't typically work in the Latin codepage.

typiconman committed:
Add anchors for dotted circle and caron.alt

typiconman committed:
Update convert script and bump Copyright year

yanone commented:
The `dottedcircle` check has cleared, but `Lcaron` still used `quoteright` instead of the new `caron.alt` which is present but unused.

typiconman committed:
Fix Lcaron and spacing on U+A674

typiconman commented:
Maybe I fixed it? At least, when I run the tests locally, I don't see any problems with LCaron. I did see an issue with a non-spacing character (U+A674) that had a space, which I also removed.

yanone commented:
Okay, this looks good now, thank you.

typiconman commented:
Merged and closed.
